_posts/en/pages/2017-01-01-download.md

```diff
@@ -80,19 +80,20 @@ verify_download_checksum: "Verify that the checksum of the release file is liste
 checksum_warning_and_ok: 'In the output produced by the above command, you can safely ignore any warnings and failures, but you must ensure the output lists "$(SHASUMS_OK)" after the name of the release file you downloaded.  For example:'
 
 example_builders_line: "E777299FC265DD04793070EB944D35F9AC3DB76A Michael Ford (fanquake)"
-builder_keys_url: "https://github.com/bitcoin/bitcoin/tree/master/contrib/builder-keys"
+builder_keys_url: "https://github.com/bitcoin-core/guix.sigs/tree/main/builder-keys"
+example_builder_key_file: "fanquake.gpg"
 
 obtain_release_key: >
   <p>Bitcoin releases are signed by a number of individuals, each with a unique public
   key. In order to recognize the validity of signatures, you must use GPG to load these
   public keys locally. You can find many developer keys listed in the <a
-  href='$(BUILDER_KEYS_URL)'>bitcoin/bitcoin repository</a>, which you can then load
+  href='$(BUILDER_KEYS_URL)'>bitcoin-core/guix.sigs repository</a>, which you can then load
   into your GPG key database.</p>
 
-  <p>For example, given the <a href='$(BUILDER_KEYS_TXT_URL)'><code>
-  builder-keys/keys.txt</code></a> line
-  <code>$(EXAMPLE_BUILDERS_LINE)</code>you could load that
-  key using this command:</p>
+  <p>For example, you could load the key <a
+  href='$(BUILDER_KEYS_URL)/$(EXAMPLE_BUILDER_KEY_FILE)'><code>
+  builder-keys/$(EXAMPLE_BUILDER_KEY_FILE)</code></a> by downloading the file as <code>
```

Comment:
Being vague on purpose about downloading the file since i don't think a `curl -sS https://raw.githubusercontent.com/bitcoin-core/guix.sigs/main/builder-keys/fanquake.gpg | gpg --import` would be portable to Windows. And downloading a file should be straightforward enough?

Reply:
I think using the `raw...` link here instead of the github file viewing UI does make it a bit easier for users. From the raw URL I can right click (in firefox/macOS) and "save as..." but currently the page you see when you click here has no explicit download button.